source/_integrations/ohme.markdown

```diff
@@ -143,6 +143,47 @@ The `ohme.set_price_cap` action is used to set the price cap threshold. This can
 | `config_entry`         | No       | The config entry of the account to apply the price cap to.  |
 | `price_cap`            | No       | Threshold in 1/100ths of your local currency.               |
 
+
+## Use cases
+
+This integration can be used to show charger status on your dashboards. It can also be used as part of automations for solar and battery storage.
+
+
+## Examples
+
+### Send a notification when a vehicle is plugged in:
```

Comment:
- Titles shouldn't contain a colon (`:`)
- Needs a new line after the title
- Missing introduction in the paragraph